content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/cpp.md

```diff
@@ -23,153 +23,169 @@ further_reading:
   tag: "Advanced Usage"
   text: "Advanced Usage"
 ---
-**Note**: C++ does not provide integrations for OOTB instrumentation, but it's used by Proxy tracing such as [Envoy][1] and [Nginx][2]. For compatibility requirements for the C++ Tracer, visit the [Compatibility Requirements][3] page.
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> C++ does not provide integrations for automatic instrumentation, but it's used by Proxy tracing such as <a href="/tracing/setup/envoy/">Envoy</a> and <a href="/tracing/setup/nginx/">Nginx</a>.
+</div>
+
+## Compatibility requirements
+The C++ tracing library requires C++17 toolchain to build. For a full list of Datadog's tracing library compatiblity requirements and processor architecture support, visit the [Compatibility Requirements][3] page.
+
+## Getting started
+Before you begin, make sure you have already [installed and configured the Agent][6].
 
 ## Instrument your application
 
-After [the Agent is installed][6], follow these steps to add the Datadog tracing library to your C++ applications in one of two ways:
+{{< tabs >}}
 
-* Compile against dd-opentracing-cpp, where the Datadog lib is compiled in and configured in code
-* Dynamic loading, where the Datadog OpenTracing library is loaded at runtime and configured via JSON
+{{% tab "CMake" %}}
```

Comment:
```suggestion
{{% tab "CMake" %}}
To integrate the `dd-trace-cpp` library into your C++ project using CMake, follow these steps:
```
^ I think we add more introduction to these code examples so it's clear for users. Something like this (please check).